cfme/tests/configure/test_access_control.py

```diff
@@ -695,9 +695,54 @@ def test_superadmin_tenant_crud():
     tenant = ac.Tenant(
         name='tenant1' + fauxfactory.gen_alphanumeric(),
         description='tenant1 description')
+
+    @request.addfinalizer
+    def _delete_tenant_and_project():
+        if tenant.exists:
+            tenant.delete()
+
     tenant.create()
     with update(tenant):
         tenant.description = tenant.description + "edited"
     with update(tenant):
         tenant.name = tenant.name + "edited"
     tenant.delete()
+
+
+@pytest.mark.uncollectif(lambda: version.current_version() < "5.5")
+def test_superadmin_tenant_project_crud(request):
+    """Test suppose to verify CRUD operations for CFME projects
+
+    Prerequisities:
+        * This test is not depending on any other test and can be executed against fresh appliance.
+
+    Steps:
+        * Create tenant
+        * Create project as child to tenant
+        * Update description of project
+        * Update name of project
+        * Delete project
+        * Delete tenant
+    """
+    tenant = ac.Tenant(
+        name='tenant1' + fauxfactory.gen_alphanumeric(),
+        description='tenant1 description')
+    project = ac.Project(
+        name='project1' + fauxfactory.gen_alphanumeric(),
+        description='project1 description',
+        parent_tenant=ac.Tenant(tenant.name))
+
+    @request.addfinalizer
+    def _delete_tenant_and_project():
+        for item in [tenant, project]:
```

Comment:
You want to do this in reverse order, since project is a child of a tenant so it would probably not allow you to delete the tenant first. (just guessing).